field/field_GF2_128.cpp

```diff
@@ -235,6 +235,49 @@ bool GF2_128::operator!=(const GF2_128 &other) const {
 }
 
 GF2_128 GF2_128::inverse() const {
+  size_t u[11] = {1, 2, 3, 6, 12, 24, 48, 51, 63, 126, 127};
```

Comment:
This can probably be const
```suggestion
  constexpr size_t u[11] = {1, 2, 3, 6, 12, 24, 48, 51, 63, 126, 127};
```